frontend/survey/surveyDetailsDirective.js

```diff
@@ -40,7 +40,13 @@
 
         surveyCtrl.isSurveyAuthor = function isSurveyAuthor() {
             return surveyCtrl.post.author_key == surveyCtrl.user.key;
-        }
+        };
+
+        surveyCtrl.timeHasBeenExpired = function timeHasBeenExpired() {
+            var nationalTimeZone = new Date (surveyCtrl.post.deadline);
+            nationalTimeZone.setHours(nationalTimeZone.getHours() - 3);
```

Comment:
That subtraction to get a timezone can cause errors, once that 3 is not valid to all users, think how that is gone to work if the user its placed outside of Brazil, or in the -4 Brazilian timezone (acre). You should work with momentJs, that we already have imported.